File: Code/AllOut.Api/AllOut.Api/DataAccess/Models/Brand.cs
using System.ComponentModel.DataAnnotations;

namespace AllOut.Api.DataAccess.Models
{
    public class Brand
    {
        [Key]
        public Guid BrandID { get; set; }
        [Required, MaxLength(50)]
        public string Name { get; set; }
        [MaxLength(100)]
        public string? Description { get; set; }
        public int Status { get; set; } 
        public DateTime CreatedDate { get; set; }
        public DateTime? ModifiedDate { get; set; }
    }
}

File: web/app/Models/Brand.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Http\Request;

class Brand extends Model
{
    use HasFactory;

    public const GET_ALL_DATA = '/Brand/GetBrands?clientID=';
    public const GET_DATA_BY_ID = '/Brand/GetBrandByID?clientID=';
    public const GET_DATA_BY_QUERY = '/Brand/GetBrandsByQuery?clientID=';
    public const STORE_DATA = '/Brand/SaveBrand';
    public const RETRIVE_DATA_BY_ID = '/Brand/GetBrandByID?clientID=';


    public function storeRequest($storeData) : array {
        return [
            'functionID' => $storeData['functionID'],
            'requestStatus' => 'A2',
            'client' => session('loginCredentials'),
            'inputBrand' => [
                'brandID' => isset($storeData['brandID']) ? $storeData['brandID'] : '00000000-0000-0000-0000-000000000000',
                'name' => $storeData['name'],
                'description' => $storeData['description'],
                'status' => 0,
                'createdDate' =>  isset($storeData['createdDate']) ? $storeData['createdDate'] : date('Y-m-d'),
                'modifiedDate' =>  isset($storeData['modifiedDate']) ? $storeData['modifiedDate'] : null,
            ]
        ];
    }

}
